### PR TITLE
ci cleanups

### PR DESCRIPTION
**Pull request by sbcgua**
to #2916, cleanups part
auto tagging will follow

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,16 @@ node_js:
   - "10"
 notifications:
   email: false
+script:
+  - npm run eslint
+  - npm run lint
```
Comment (larshp) [inline .travis.yml]: rename this one to abaplint?

Comment (sbcgua) [reply .travis.yml]: done

**Comment by sbcgua** [comment]
let's merge maybe ?

**Comment by larshp** [comment]
I dont really see what it cleans up, anyhow 🤷‍♂

**Pull request merged**